### ISSUE TITLE
how to evaluate the model on the test dataset?

### ISSUE THREAD

**Issue by MsRabbits** [fizyr/keras-retinanet#1134]
Please make sure that you follow the steps below when creating an issue.
Only use GitHub issues for issues with the implementation, not for issues with specific datasets or general questions about functionality.
If your issue is an implementation question, please ask your question on the #keras-retinanet Slack channel instead of filing a GitHub issue.
You can find directions for the Slack channel here: https://github.com/fizyr/keras-retinanet#discussions

Thank you!

1. Check that you are up-to-date with the master branch of keras-retinanet.
2. Check that you are up-to-date with the latest version of Keras: https://github.com/keras-team/keras.
3. Check that you are up-to-date with the latest version of TensorFlow.
   The installation instructions can be found here: https://www.tensorflow.org/get_started/os_setup.
4. Check that you have read the entire README.md: https://github.com/fizyr/keras-retinanet/README.md.
   Most noticably the FAQ section shows common issues: https://github.com/fizyr/keras-retinanet#faq.
5. Clearly describe the issues you're having including the expected behaviour, the actual behaviour
   and the steps required to trigger the issue.
6. Include relevant output from the commands you're executing, including full stack traces where relevant.
7. Remove this entire message and replace it with your issue.

**Comment by MsRabbits** [comment]
when  i try to evaluate the file '.h5' on the other test set, i run the  evauate.py and I have got the error:
(tensorflow) E:\pycharm2019\project_tree\objecdetection_match\keras-retinanet-master_old\keras-retinanet-master>python keras_retinanet/bin/evaluate.py pascal E:\pycharm2
019\project_tree\objecdetection_match\keras-retinanet-master_old\keras-retinanet-master\test_data
Using TensorFlow backend.
usage: evaluate.py pascal [-h] pascal_path
evaluate.py pascal: error: the following arguments are required: pascal_path
anybody help?

**Comment by MsRabbits** [comment]
i have already give the pascal_path, why there are still need 'pascal_path'?

**Comment by zjp99** [comment]
I want to test it on my data set, using my model.
How do I set the parameters? evauate.py
Using the command line

**Comment by hgaiser** [comment]
The error is a bit misleading, but you also need to provide a model to evaluate.

**Comment by MsRabbits** [comment]
> The error is a bit misleading, but you also need to provide a model to evaluate.

It seems works.THX!!!
